storm-core/test/clj/org/apache/storm/nimbus_test.clj

```diff
@@ -792,15 +795,18 @@
       (bind _ (log-message "ass1, t76, pre beat: " (pr-str ass1)))
       (bind _ (log-message "ass2, t76, pre beat: " (pr-str ass2)))
       (is (= ass1 (executor-assignment cluster storm-id executor-id1)))
+      ;TODO: fix local test of worker reports
```

Comment:
Is it addressed? If then please remove it. If not, let's have TODO in body of PR.

Reply:
I have removed it and fix the 2 TODOs already.

Reply:
I pulled the latest update of PR to local but they're still existing in nimbus_test.clj. Please check again.